library/public/camera.h

```diff
@@ -9,6 +9,14 @@
 
 namespace f3d
 {
+struct F3D_EXPORT camera_state_t
+{
+  point3_t pos = { 0, 0, 1 };
+  point3_t foc = { 0, 0, 0 };
+  vector3_t up = { 0, 1, 0 };
+  angle_deg_t angle = 30;
```

Comment:
@snoyer @Meakk so let me know what yhou think of this init. imo it is fine.

Reply:
it's a double, so `30.0`? And similar for the values of the vectors.